docs/source/ops_guide.rst

```diff
@@ -4,6 +4,7 @@ Operations Guides
 .. toctree::
    :maxdepth: 1
 
+   use_CA
    orderer_deploy
```

Comment:
What is the use_CA link here? I assume it is for the Using a CA topic in the Fabric CA repo that has not been merged yet?

Reply:
It's a mistake --- something that was hanging around from something old and shouldn't have been in there. There were enough docs in the PR I didn't see it.